test/cmdlineTests/yul_string_format_ascii/output.json

```diff
@@ -38,6 +38,7 @@ object \"C_10\" {
             let zero_value_for_type_t_string_memory_ptr_1_mpos := zero_value_for_split_t_string_memory_ptr()
             vloc__4_mpos := zero_value_for_type_t_string_memory_ptr_1_mpos
 
+
```

Comment:
I'm not sure if empty lines are ignored when the output is compared in tests but if they are then I would suggest removing this and other similar changes. Previously you had other changes here so it didn't matter as much but now that they were removed, the whitespace changes are the only changes here and they're just noise.

Reply:
Actually it needs to be here so test are passing. I planned to check which part of code adds those lines and try fixing.

Reply:
Oh, so they're not ignored :) OK then. It would be better not to have them but I guess your time is better spent elsewhere.